android/src/main/kotlin/co/deeprooted/plugin/juspay/juspay_flutter/JuspayFlutterPlugin.kt

```diff
@@ -137,4 +138,9 @@ class JuspayFlutterPlugin: FlutterPlugin, MethodCallHandler, ActivityAware, Plug
         hyperServices!!.terminate()
         result.success(true)
     }
+
+    private fun onBackPressed() {
+        hyperServices!!.onBackPressed()
```

Comment:
Can you add `result.success(true)`

Reply:
Also, possibly a try/catch around `onBackPressed` in case it throws an exception